docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -643,6 +643,34 @@ but you can still access the original via ``hostvars``::
 This works for all overriden connection variables, like ``ansible_user``, ``ansible_port``, etc.
 
 
+.. _scp_protocol_error_filename:
+
+How do I fix 'protocol error: filename does not match request' when fetching a file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+When using ``scp`` as the file transfer mechanism for SSH, it can fail with the error::
+
+    failed to transfer file to /tmp/ansible/file.txt\r\nprotocol error: filename does not match request
+
+This is due to a bug in newer releases of OpenSSH in the ``scp`` client that is used on the Ansible controller. These
```

Comment:
Once we move the reason up, this section can be slimmed down: "The affected OpenSSH releases attempt to . . ."